<!DOCTYPE html>
<html>
<head>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
<meta http-equiv="x-ua-compatible" content="IE=edge"/>
<title>Test results - Class itrs_appserver.UserControllerTest</title>
<link href="../css/base-style.css" rel="stylesheet" type="text/css"/>
<link href="../css/style.css" rel="stylesheet" type="text/css"/>
<script src="../js/report.js" type="text/javascript"></script>
</head>
<body>
<div id="content">
<h1>Class itrs_appserver.UserControllerTest</h1>
<div class="breadcrumbs">
<a href="../index.html">all</a> &gt; 
<a href="../packages/itrs_appserver.html">itrs_appserver</a> &gt; UserControllerTest</div>
<div id="summary">
<table>
<tr>
<td>
<div class="summaryGroup">
<table>
<tr>
<td>
<div class="infoBox" id="tests">
<div class="counter">11</div>
<p>tests</p>
</div>
</td>
<td>
<div class="infoBox" id="failures">
<div class="counter">0</div>
<p>failures</p>
</div>
</td>
<td>
<div class="infoBox" id="ignored">
<div class="counter">0</div>
<p>ignored</p>
</div>
</td>
<td>
<div class="infoBox" id="duration">
<div class="counter">1.002s</div>
<p>duration</p>
</div>
</td>
</tr>
</table>
</div>
</td>
<td>
<div class="infoBox success" id="successRate">
<div class="percent">100%</div>
<p>successful</p>
</div>
</td>
</tr>
</table>
</div>
<div id="tabs">
<ul class="tabLinks">
<li>
<a href="#tab0">Tests</a>
</li>
<li>
<a href="#tab1">Standard output</a>
</li>
<li>
<a href="#tab2">Standard error</a>
</li>
</ul>
<div id="tab0" class="tab">
<h2>Tests</h2>
<table>
<thead>
<tr>
<th>Test</th>
<th>Duration</th>
<th>Result</th>
</tr>
</thead>
<tr>
<td class="success">checkLoggedInStatusTest</td>
<td>0s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginAuthotrisedDomainFirstTimeTest</td>
<td>0.243s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginAuthotrisedDomainSecondTimeWithSecondDeviceTest</td>
<td>0.453s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginAuthotrisedDomainSecondTimeWithoutVeriticationTest</td>
<td>0.074s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginAuthotrisedUserWithFirstDeviceTest</td>
<td>0.048s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginAuthotrisedUserWithSecondDeviceTest</td>
<td>0.058s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginFailureCatchTest</td>
<td>0.008s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">loginUnauthorisedDomainTest</td>
<td>0.016s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">logoutTest</td>
<td>0.060s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">setCustomDVTestFailTest</td>
<td>0.001s</td>
<td class="success">passed</td>
</tr>
<tr>
<td class="success">setCustomDVTestSucceedTest</td>
<td>0.041s</td>
<td class="success">passed</td>
</tr>
</table>
</div>
<div id="tab1" class="tab">
<h2>Standard output</h2>
<span class="code">
<pre>16:45:18.295 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-16] INFO  c.itrsgroup.actor.MessageDispatcher - Successfully connected to remote service [akka.tcp://ClusterSystem@192.168.220.54:2551/user/frontEnd]
16:45:18.296 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-16] INFO  c.itrsgroup.actor.MessageDispatcher - Forwarding Subscription op to cluster frontend: Subscribe(ModelSub(testa10/rows/row[wild(@name,&quot;*&quot;)]/cell,Set(),LIST,),None)
16:45:18.395 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-9] INFO  c.itrsgroup.actor.MessageDispatcher - Successfully connected to remote service [akka.tcp://ClusterSystem@192.168.220.54:2551/user/frontEnd]
16:45:18.395 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-9] INFO  c.itrsgroup.actor.MessageDispatcher - Forwarding Subscription op to cluster frontend: Subscribe(ModelSub(testa9/rows/row[wild(@name,&quot;*&quot;)]/cell,Set(),LIST,),None)
16:45:18.435 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-3] ERROR c.itrsgroup.actor.DataSetClientActor - Exception calling error callback
java.lang.RuntimeException: null
	at geneos_notification.thread_operations.ThreadInstance$2.error(ThreadInstance.java:111) ~[main/:na]
	at com.itrsgroup.actor.ClientActor$$anonfun$1.apply$mcV$sp(ClientActor.scala:143) ~[geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor$$anonfun$1.apply(ClientActor.scala:143) ~[geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor$$anonfun$1.apply(ClientActor.scala:143) ~[geneos-openaccess-client.jar:2.0.1]
	at scala.util.Try$.apply(Try.scala:191) ~[geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor.fireError(ClientActor.scala:143) [geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor$$anonfun$receive$1.applyOrElse(ClientActor.scala:84) [geneos-openaccess-client.jar:2.0.1]
	at akka.actor.Actor$class.aroundReceive(Actor.scala:465) [geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor.aroundReceive(ClientActor.scala:38) [geneos-openaccess-client.jar:2.0.1]
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:516) [geneos-openaccess-client.jar:2.0.1]
	at akka.actor.ActorCell.invoke(ActorCell.scala:487) [geneos-openaccess-client.jar:2.0.1]
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:254) [geneos-openaccess-client.jar:2.0.1]
	at akka.dispatch.Mailbox.run(Mailbox.scala:221) [geneos-openaccess-client.jar:2.0.1]
	at akka.dispatch.Mailbox.exec(Mailbox.scala:231) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.pollAndExecAll(ForkJoinPool.java:1253) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1346) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [geneos-openaccess-client.jar:2.0.1]
16:45:18.438 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-10] ERROR c.itrsgroup.actor.DataSetClientActor - Exception calling error callback
java.lang.RuntimeException: null
	at geneos_notification.thread_operations.ThreadInstance$2.error(ThreadInstance.java:111) ~[main/:na]
	at com.itrsgroup.actor.ClientActor$$anonfun$1.apply$mcV$sp(ClientActor.scala:143) ~[geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor$$anonfun$1.apply(ClientActor.scala:143) ~[geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor$$anonfun$1.apply(ClientActor.scala:143) ~[geneos-openaccess-client.jar:2.0.1]
	at scala.util.Try$.apply(Try.scala:191) ~[geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor.fireError(ClientActor.scala:143) [geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor$$anonfun$receive$1.applyOrElse(ClientActor.scala:84) [geneos-openaccess-client.jar:2.0.1]
	at akka.actor.Actor$class.aroundReceive(Actor.scala:465) [geneos-openaccess-client.jar:2.0.1]
	at com.itrsgroup.actor.ClientActor.aroundReceive(ClientActor.scala:38) [geneos-openaccess-client.jar:2.0.1]
	at akka.actor.ActorCell.receiveMessage(ActorCell.scala:516) [geneos-openaccess-client.jar:2.0.1]
	at akka.actor.ActorCell.invoke(ActorCell.scala:487) [geneos-openaccess-client.jar:2.0.1]
	at akka.dispatch.Mailbox.processMailbox(Mailbox.scala:254) [geneos-openaccess-client.jar:2.0.1]
	at akka.dispatch.Mailbox.run(Mailbox.scala:221) [geneos-openaccess-client.jar:2.0.1]
	at akka.dispatch.Mailbox.exec(Mailbox.scala:231) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.pollAndExecAll(ForkJoinPool.java:1253) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1346) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979) [geneos-openaccess-client.jar:2.0.1]
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107) [geneos-openaccess-client.jar:2.0.1]
16:45:19.068 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-4] INFO  akka.event.slf4j.Slf4jLogger - Slf4jLogger started
16:45:19.071 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-4] INFO  Remoting - Starting remoting
16:45:19.082 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-4] INFO  Remoting - Remoting started; listening on addresses :[akka.tcp://OpenAccessApiClient-UnknownDevBuild@192.168.10.128:10921]
[teste]
16:45:19.083 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-9] INFO  c.itrsgroup.actor.MessageDispatcher - Attempting to connect to remote service at [192.168.220.54:2551] 1 time(s)
16:45:19.108 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-5] INFO  akka.contrib.pattern.ClusterClient - Connected to [akka.tcp://ClusterSystem@192.168.220.54:2551/user/receptionist]
16:45:19.114 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-9] INFO  c.itrsgroup.actor.MessageDispatcher - Authenticating with remote service [/user/frontEnd] at [192.168.220.54:2551]
16:45:19.240 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-8] INFO  c.itrsgroup.actor.MessageDispatcher - Successfully connected to remote service [akka.tcp://ClusterSystem@192.168.220.54:2551/user/frontEnd]
16:45:19.240 [OpenAccessApiClient-UnknownDevBuild-akka.actor.default-dispatcher-8] INFO  c.itrsgroup.actor.MessageDispatcher - Forwarding Subscription op to cluster frontend: Subscribe(ModelSub(test/rows/row[wild(@name,&quot;*&quot;)]/cell,Set(),LIST,),None)
</pre>
</span>
</div>
<div id="tab2" class="tab">
<h2>Standard error</h2>
<span class="code">
<pre>Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testb@Default
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testb@Default &amp; testb
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
SEVERE: [T-ERROR]Error retrieving DataSet: java.lang.Exception: Invalid step in path: testa10
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
WARNING: [T-WARNING]OpenAccess null response due to multiple reloads, this is handled and expected. Staggering in progress.
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
SEVERE: [T-ERROR]Error retrieving DataSet: java.lang.Exception: Invalid step in path: testa9
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
WARNING: [T-WARNING]OpenAccess null response due to multiple reloads, this is handled and expected. Staggering in progress.
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]New user testb@Default has attempted to login, verification email sent and account is in probation.
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testb@Default
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testb@Default &amp; testz
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device Check, User has yet to verify device : testz
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]testb@Default has been sent a verification email for testz
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : test@Default
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : test@Default &amp; test
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]New user test@Default has attempted to login, verification email sent and account is in probation.
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : fail' or 1 = 1
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : fail' or 1 = 1 &amp; fail' or 1 = 1
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
SEVERE: [SQL]SQL query issue was encountered causing SQL failure, Usercheck failure: com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''' at line 1
java.lang.RuntimeException: com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''' at line 1
	at geneos_notification.controllers.DatabaseController.checkUser(DatabaseController.java:75)
	at geneos_notification.controllers.UserController.login(UserController.java:27)
	at itrs_appserver.UserControllerTest.loginFailureCatchTest(UserControllerTest.java:102)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:50)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:47)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:78)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:57)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:86)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:49)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:64)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.messaging.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.messaging.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy2.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:106)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.messaging.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.messaging.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:360)
	at org.gradle.internal.concurrent.DefaultExecutorFactory$StoppableExecutorImpl$1.run(DefaultExecutorFactory.java:64)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
Caused by: com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''' at line 1
	at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:936)
	at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:2985)
	at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:1631)
	at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:1723)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3277)
	at com.mysql.jdbc.Connection.execSQL(Connection.java:3206)
	at com.mysql.jdbc.Statement.executeQuery(Statement.java:1232)
	at geneos_notification.controllers.DatabaseController.checkUser(DatabaseController.java:58)
	... 46 more
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
SEVERE: [U-Controller]Login error has occured, please see associated stack trace for more information. Username : fail' or 1 = 1 android_id : fail' or 1 = 1
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: java.lang.RuntimeException: com.mysql.jdbc.exceptions.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''' at line 1
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testf@Default
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testf@Default &amp; testf
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testf logged in successfully
Mar 17, 2016 4:45:18 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testf key has been updated
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Initial log on for testf@Default with device testf
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testf logged out successfully
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]The status of the device check fortestf@Default is :1
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]testf@Default has been successfully logged out.
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : teste@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : teste@Default &amp; teste
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device teste logged in successfully
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device teste key has been updated
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Initial log on for teste@Default with device teste
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : something@wrong
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : something@wrong &amp; test
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
WARNING: [SQL]User has tried to log in with an invalid domain : something@wrong
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
WARNING: [U-Controller]Attempted login made by unauthorised user with username : something@wrong
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testc@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testc@Default &amp; testc
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testc logged in successfully
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testc key has been updated
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Initial log on for testc@Default with device testc
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
SEVERE: [U-Controller]Attempted access by invalid and potentially harmful user utilising username : testFail@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testd@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testd@Default &amp; testd
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testd logged in successfully
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testd key has been updated
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Initial log on for testd@Default with device testd
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testd@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testd@Default &amp; testx
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testx logged in successfully
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device testx key has been updated
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]testd@Default logged in successfully on device testx
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testa@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testa@Default &amp; testa
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]New user testa@Default has attempted to login, verification email sent and account is in probation.
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]Login attempt with username : testa@Default
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Checking user : testa@Default &amp; testa
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [SQL]Device Check, User has yet to verify device : testa
Mar 17, 2016 4:45:19 PM geneos_notification.loggers.LogObject doLog
INFO: [U-Controller]testa@Default has already been sent a verification email for testa
</pre>
</span>
</div>
</div>
<div id="footer">
<p>
<div>
<label class="hidden" id="label-for-line-wrapping-toggle" for="line-wrapping-toggle">Wrap lines
<input id="line-wrapping-toggle" type="checkbox" autocomplete="off"/>
</label>
</div>Generated by 
<a href="http://www.gradle.org">Gradle 2.3</a> at 17-Mar-2016 16:45:19</p>
</div>
</div>
</body>
</html>
